apps/meteor/app/lib/server/methods/getChannelHistory.ts

```diff
@@ -39,12 +39,12 @@ Meteor.methods({
 		}
 
 		// Ensure latest is always defined.
-		if (_.isUndefined(latest)) {
+		if (latest === undefined) {
 			latest = new Date();
 		}
 
 		// Verify oldest is a date if it exists
-		if (!_.isUndefined(oldest) && !_.isDate(oldest)) {
+		if (oldest !== undefined && !_.isDate(oldest)) {
```

Comment:
Can we use Date.parse to check?

Reply:
`_.isDate()` is a type guard that uses `Object.prototype.toString` to validate the object type, while `Date.parse` will try to coerce a string whenever possible. Very different.